File: gin/entity/po/collect.go
package po

import "time"

type Collect struct {
	ID        uint      `gorm:"column:id" json:"id"`
	ArticleID uint      `gorm:"column:article_id" json:"articleId"`
	UserID    uint      `gorm:"column:user_id" json:"userId"`
	CreatedAt time.Time `gorm:"column:created_time" json:"createdAt"`
}

func (Collect) TableName() string {
	return "collects"
}

File: fastapi/app/internal/models/collect.py
from datetime import datetime

from app.core.db.base import Base
from sqlalchemy import Column, DateTime, Integer


class Collect(Base):
    """收藏实体类"""

    __tablename__ = "collects"
    id = Column(Integer, primary_key=True, index=True, autoincrement=True)
    article_id = Column(Integer, nullable=False, index=True)
    user_id = Column(Integer, nullable=False, index=True)
    created_time = Column(DateTime, nullable=True, default=datetime.utcnow)
